lib/gollum/app.rb

```diff
@@ -639,6 +636,11 @@ def show_file(file)
       end
     end
 
+    def load_template(path)
+      template_page = wiki_page(::File.join(path, '_Template')).page || wiki_page('/_Template').page
+      template_page ? Gollum::TemplateFilter.apply_filters(template_page.raw_data) : nil
```

Comment:
@bartkamphorst @dometto How about passing `path` to `apply_filters` so that it could be accessed in the `replacement`?

This makes a filter like `{{page_name}}` possible.

Reply:
This makes sense to me, if you're wiling to open a PR for this @tevino! What do you think @bartkamphorst?

Reply:
@dometto I just submitted a PR #1818, please help to review it, thanks!